docs/reference/manifest/runtimes.md

```diff
@@ -1,12 +1,15 @@
 ---
 title: Runtimes in the manifest file 
 description: The Runtimes element specifies your add-in's runtime.
-ms.date: 05/29/2020
+ms.date: 06/01/2020
 localization_priority: Normal
 ---
 # Runtimes element
 
-Specifies the runtime of your add-in. Child of the [`<Host>`](host.md) element. If the `Runtimes` element is present in your manifest, your add-in will by default use the Internet Explorer 11 browser.
+Specifies the runtime of your add-in. Child of the [`<Host>`](host.md) element.
+
+> [!NOTE]
+> When running in a Windows app, your add-in will use the Internet Explorer 11 browser by default.
```

Comment:
Does "Windows app" here mean "Office on Windows"? If not, what does it mean?

Reply:
Yes, that's the case. I'll clarify that. I didn't want to simply say "on Windows" because I didn't want ppl to think it included using Outlook or Excel in a browser on Windows.